#VRML_SIM R2018b utf8
WorldInfo {
  contactProperties [
    ContactProperties {
      material1 "castorBall"
      coulombFriction [
        0
      ]
    }
  ]
}
Viewpoint {
  orientation -0.10746678148653059 0.990107146984454 0.0902149010265554 1.6218728433319418
  position 5.171308343534414 0.2621093312882876 0.005604335939319492
  follow "robot"
  followOrientation TRUE
}
DirectionalLight {
  direction 0 -1 0
}
RectangleArena {
  floorSize 10 5
}
Solid {
  translation -2.334523855692977 0.0699999999997642 -0.6469357705279251
  children [
    DEF sphere Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Sphere {
        radius 0.14
      }
    }
  ]
  name "solid(45)"
  boundingObject USE sphere
}
Background {
  skyColor [
    0.4470588235294118 0.6235294117647059 0.8117647058823529
  ]
}
Robot {
  translation 4.583308870007387 0.004708270651314914 0.024500815486374977
  rotation 0 0.9999999999999999 0 4.712388966
  children [
    HingeJoint {
      jointParameters HingeJointParameters {
        anchor -0.056 0.03 0
      }
      device [
        PositionSensor {
          name "positionSensorLeft"
        }
        RotationalMotor {
          name "rotationalMotorLeft"
          maxTorque 100
        }
      ]
      endPoint Solid {
        translation -0.056 0.03 0
        rotation -5.890192445936235e-17 9.816987409893724e-18 1 1.57
        children [
          Transform {
            children [
              Shape {
                appearance Appearance {
                  material Material {
                    diffuseColor 0 0 0
                  }
                }
                geometry DEF wheel Cylinder {
                  height 0.01
                  radius 0.025
                }
              }
            ]
          }
        ]
        boundingObject USE wheel
        physics Physics {
        }
      }
    }
    HingeJoint {
      jointParameters HingeJointParameters {
        anchor 0.056 0.03 0
      }
      device [
        PositionSensor {
          name "positionSensorRight"
        }
        RotationalMotor {
          name "rotationalMotorRight"
          maxTorque 100
        }
      ]
      endPoint Solid {
        translation 0.056 0.03 0
        rotation -0.7073867688153652 0.7068236819478265 0.0020595976973216495 3.1374701825011044
        children [
          Transform {
            children [
              Shape {
                appearance Appearance {
                  material Material {
                    diffuseColor 0 0 0
                  }
                }
                geometry Cylinder {
                  height 0.01
                  radius 0.025
                }
              }
            ]
          }
        ]
        name "solid(1)"
        boundingObject USE wheel
        physics Physics {
        }
      }
    }
    Transform {
      translation 0 0.03 0
      children [
        Shape {
          appearance Appearance {
            material Material {
              diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
            }
          }
          geometry Box {
            size 0.06 0.02 0.09
          }
        }
      ]
    }
    Transform {
      translation 0 0.05 0
      children [
        Shape {
          appearance Appearance {
            material Material {
              diffuseColor 0.3333333333333333 0.3411764705882353 0.3254901960784314
            }
          }
          geometry Cylinder {
            height 0.02
            radius 0.05
          }
        }
      ]
    }
    Transform {
      translation 0 0.03 0
      rotation 0 0 1 1.5707963220000003
      children [
        Shape {
          appearance Appearance {
            material Material {
              diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
            }
          }
          geometry Cylinder {
            height 0.11
            radius 0.005
          }
        }
      ]
    }
    DistanceSensor {
      translation 0 0.07 0.03
      rotation 0 0.9999999999999999 0 4.712388966
      children [
        Transform {
          translation 0.008 0 0.01
          rotation -0.5773502627301778 -0.5773502710351824 0.5773502738035171 4.188790199249721
          children [
            Shape {
              appearance Appearance {
                material Material {
                }
              }
              geometry Cylinder {
                height 0.013
                radius 0.008
              }
            }
          ]
        }
        Transform {
          translation 0.008 0 -0.01
          rotation -0.5773502627301778 -0.5773502710351824 0.5773502738035171 4.188790199249721
          children [
            Shape {
              appearance Appearance {
                material Material {
                }
              }
              geometry Cylinder {
                height 0.013
                radius 0.008
              }
            }
          ]
        }
        Transform {
          rotation 0 1 0 1.5707963220000003
          children [
            Shape {
              appearance Appearance {
                material Material {
                }
              }
              geometry Box {
                size 0.04 0.02 0.003
              }
            }
          ]
        }
      ]
      name "distanceSensorCenter"
      lookupTable [
        0 0 0
        1 1000 0
      ]
    }
    DistanceSensor {
      translation 0.035 0.07 0.015
      rotation 0 -1 0 1.483529199359174
      children [
        Transform {
          translation 0.008 0 0.01
          rotation -0.5773502627301778 -0.5773502710351824 0.5773502738035171 4.188790199249721
          children [
            Shape {
              appearance Appearance {
                material Material {
                }
              }
              geometry Cylinder {
                height 0.013
                radius 0.008
              }
            }
          ]
        }
        Transform {
          translation 0.008 0 -0.01
          rotation -0.5773502627301778 -0.5773502710351824 0.5773502738035171 4.188790199249721
          children [
            Shape {
              appearance Appearance {
                material Material {
                }
              }
              geometry Cylinder {
                height 0.013
                radius 0.008
              }
            }
          ]
        }
        Transform {
          rotation 0 1 0 1.5707963220000003
          children [
            Shape {
              appearance Appearance {
                material Material {
                }
              }
              geometry Box {
                size 0.04 0.02 0.003
              }
            }
          ]
        }
      ]
      name "distanceSensorLeft"
      lookupTable [
        0 0 0
        1 1000 0
      ]
    }
    DistanceSensor {
      translation -0.035 0.07 0.015
      rotation 0 1 0 4.624787126999999
      children [
        Transform {
          translation 0.008 0 0.01
          rotation -0.5773502627301778 -0.5773502710351824 0.5773502738035171 4.188790199249721
          children [
            Shape {
              appearance Appearance {
                material Material {
                }
              }
              geometry Cylinder {
                height 0.013
                radius 0.008
              }
            }
          ]
        }
        Transform {
          translation 0.008 0 -0.01
          rotation -0.5773502627301778 -0.5773502710351824 0.5773502738035171 4.188790199249721
          children [
            Shape {
              appearance Appearance {
                material Material {
                }
              }
              geometry Cylinder {
                height 0.013
                radius 0.008
              }
            }
          ]
        }
        Transform {
          rotation 0 1 0 1.5707963220000003
          children [
            Shape {
              appearance Appearance {
                material Material {
                }
              }
              geometry Box {
                size 0.04 0.02 0.003
              }
            }
          ]
        }
      ]
      name "distanceSensorRight"
      lookupTable [
        0 0 0
        1 1000 0
      ]
    }
    Transform {
      translation 0 0.07 -0.01
      children [
        Shape {
          appearance Appearance {
            material Material {
              diffuseColor 0.7294117647058823 0.7411764705882353 0.7137254901960784
            }
          }
          geometry Box {
            size 0.05 0.02 0.065
          }
        }
      ]
    }
    Solid {
      translation 0 0.014 0.035
      children [
        DEF castor Shape {
          appearance Appearance {
            material Material {
            }
          }
          geometry Sphere {
            radius 0.01
          }
        }
      ]
      name "solid(2)"
      contactMaterial "castorBall"
      boundingObject USE castor
      physics Physics {
      }
    }
    Compass {
      rotation 0 1 0 3.141592644
    }
  ]
  boundingObject Transform {
    translation 0 0.0505 0
    children [
      Cylinder {
        height 0.0885
        radius 0.05
      }
    ]
  }
  physics Physics {
  }
  controller "AnalogRobotController"
}
Solid {
  translation 3.3599800939347073 0.14999999999999866 0.621718065134744
  children [
    DEF cube Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Box {
        size 0.3 0.3 0.3
      }
    }
  ]
  boundingObject USE cube
}
Solid {
  translation 2.226508235007085 0.14999999999999955 0.9596646237364166
  children [
    DEF cube Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Box {
        size 0.3 0.3 0.3
      }
    }
  ]
  name "solid(36)"
  boundingObject USE cube
}
Solid {
  translation -3.0859123314881423 0.14999999999999866 1.6809760606541346
  children [
    DEF cube Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Box {
        size 0.3 0.3 0.3
      }
    }
  ]
  name "solid(5)"
  boundingObject USE cube
}
Solid {
  translation 1.2451322288480888 0.1499999999999991 1.224913640253234
  children [
    DEF cube Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Box {
        size 0.3 0.3 0.3
      }
    }
  ]
  name "solid(7)"
  boundingObject USE cube
}
Solid {
  translation -1.9420410505138799 0.1499999999999911 -1.4461420003463237
  children [
    DEF cube Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Box {
        size 0.3 0.3 0.3
      }
    }
  ]
  name "solid(2)"
  boundingObject USE cube
}
Solid {
  translation 1.7541473689322151 0.15 -1.141722354489875
  children [
    DEF cube Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Box {
        size 0.3 0.3 0.3
      }
    }
  ]
  name "solid(42)"
  boundingObject USE cube
}
Solid {
  translation -3.0159410944736784 0.1499999999999991 -1.6047896165695508
  children [
    DEF cube Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Box {
        size 0.3 0.3 0.3
      }
    }
  ]
  name "solid(3)"
  boundingObject USE cube
}
Solid {
  translation 3.8798525061657445 0.14999999999999644 -1.306834489648518
  children [
    DEF cube Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Box {
        size 0.3 0.3 0.3
      }
    }
  ]
  name "solid(4)"
  boundingObject USE cube
}
Solid {
  translation -1.6729947508731549 0.14999999999999777 0.4302711545960327
  children [
    DEF cube Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Box {
        size 0.3 0.3 0.3
      }
    }
  ]
  name "solid(6)"
  boundingObject USE cube
}
Solid {
  translation 0 0.15 0
  children [
    DEF cube Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Box {
        size 0.3 0.3 0.3
      }
    }
  ]
  name "solid(1)"
  boundingObject USE cube
}
Solid {
  translation -3.2129449307593916 0.125 -0.7474746250640856
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(8)"
  boundingObject USE cyl
}
Solid {
  translation -1.5893494055450255 0.125 1.767558673306223
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(12)"
  boundingObject USE cyl
}
Solid {
  translation 2.0197345202465966 0.125 -0.07027036791858099
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(14)"
  boundingObject USE cyl
}
Solid {
  translation -4.4751868222293965 0.12499999999999423 -0.15097972567472095
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(16)"
  boundingObject USE cyl
}
Solid {
  translation 1.7858264369395949 0.12500000000000178 1.8838018800073983
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(18)"
  boundingObject USE cyl
}
Solid {
  translation 4.4213383892978895 0.12499999999999822 0.798626958221814
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(19)"
  boundingObject USE cyl
}
Solid {
  translation 1.0269679775778897 0.125 -1.8623624181973484
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(20)"
  boundingObject USE cyl
}
Solid {
  translation -2.8373159919785014 0.12500000000000044 1.0828220322531488
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(22)"
  boundingObject USE cyl
}
Solid {
  translation -0.1058725310483748 0.12499999999999822 1.598865370575527
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(24)"
  boundingObject USE cyl
}
Solid {
  translation 3.8480965237151503 0.12499999999999645 1.755354561584815
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(9)"
  boundingObject USE cyl
}
Solid {
  translation -0.7221205429812798 0.12499999999999956 -1.4789496172063128
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(10)"
  boundingObject USE cyl
}
Solid {
  translation -3.919737704065895 0.12500000000000178 0.7940508576679339
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(11)"
  boundingObject USE cyl
}
Solid {
  translation 3.516366462007115 0.12499999999998535 0.015284294218662953
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(13)"
  boundingObject USE cyl
}
Solid {
  translation 3.8430361769904584 0.12499999999999867 -0.37559751776487893
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(35)"
  boundingObject USE cyl
}
Solid {
  translation 2.6538046462266127 0.1250000000000071 -0.38105615721362884
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(15)"
  boundingObject USE cyl
}
Solid {
  translation -1.1300636807208324 0.12500000000000622 -0.1035917103622428
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(40)"
  boundingObject USE cyl
}
Solid {
  translation 2.605201281762942 0.12500000000001332 0.25099877899107437
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(39)"
  boundingObject USE cyl
}
Solid {
  translation 2.538160648180728 0.12500000000000444 -1.9450520990807494
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(41)"
  boundingObject USE cyl
}
Solid {
  translation -1.56540004670709 0.125 -0.7613917421190086
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(17)"
  boundingObject USE cyl
}
Solid {
  translation 0.4355215191869295 0.12499999999999867 -0.8432357509019799
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(21)"
  boundingObject USE cyl
}
Solid {
  translation -0.5171580111236622 0.12500000000005373 -0.7560810908198387
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(46)"
  boundingObject USE cyl
}
Solid {
  translation 4.414701340911306 0.125 -1.9957079673979594
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(23)"
  boundingObject USE cyl
}
Solid {
  translation 2.9215199134011325 0.07000000000000045 -0.7839823901705028
  children [
    DEF sphere Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Sphere {
        radius 0.14
      }
    }
  ]
  name "solid(25)"
  boundingObject USE sphere
}
Solid {
  translation 2.6775081891002506 0.06999999999999779 1.5143787120209382
  children [
    DEF sphere Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Sphere {
        radius 0.14
      }
    }
  ]
  name "solid(28)"
  boundingObject USE sphere
}
Solid {
  translation -1.0621778729708709 0.06999999999978551 1.4417806125259327
  children [
    DEF sphere Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Sphere {
        radius 0.14
      }
    }
  ]
  name "solid(29)"
  boundingObject USE sphere
}
Solid {
  translation -3.2290249203316943 0.06999999999978374 0.42684605663228137
  children [
    DEF sphere Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Sphere {
        radius 0.14
      }
    }
  ]
  name "solid(44)"
  boundingObject USE sphere
}
Solid {
  translation 0.5556258536335772 0.06999999999978018 0.4043054081598769
  children [
    DEF sphere Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Sphere {
        radius 0.14
      }
    }
  ]
  name "solid(43)"
  boundingObject USE sphere
}
Solid {
  translation 1.0628501654985767 0.06999999999988943 -0.5019580236203999
  children [
    DEF sphere Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Sphere {
        radius 0.14
      }
    }
  ]
  name "solid(37)"
  boundingObject USE sphere
}
Solid {
  translation 4.600975714329618 0.06999999999988854 -1.3864940849760727
  children [
    DEF sphere Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Sphere {
        radius 0.14
      }
    }
  ]
  name "solid(38)"
  boundingObject USE sphere
}
Solid {
  translation -1.7737074078325126 0.07000000000004131 1.0905860279908186
  children [
    DEF sphere Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Sphere {
        radius 0.14
      }
    }
  ]
  name "solid(32)"
  boundingObject USE sphere
}
Solid {
  translation -4.230304660776328 0.06999999999999912 2.196097801263984
  children [
    DEF sphere Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Sphere {
        radius 0.14
      }
    }
  ]
  name "solid(34)"
  boundingObject USE sphere
}
Solid {
  translation -2.789102727198247 0.06999999999999601 -0.0976095247987605
  children [
    DEF sphere Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Sphere {
        radius 0.14
      }
    }
  ]
  name "solid(27)"
  boundingObject USE sphere
}
Solid {
  translation -4.146375603386444 0.06999999999999956 -2.0709628714566133
  children [
    DEF sphere Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Sphere {
        radius 0.14
      }
    }
  ]
  name "solid(30)"
  boundingObject USE sphere
}
Solid {
  translation 0.4479980098487095 0.07000000000007772 -1.4812651823680636
  children [
    DEF sphere Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Sphere {
        radius 0.14
      }
    }
  ]
  name "solid(31)"
  boundingObject USE sphere
}
Solid {
  translation 1.8070429752889359 0.07 0.514614558115067
  children [
    DEF sphere Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Sphere {
        radius 0.14
      }
    }
  ]
  name "solid(33)"
  boundingObject USE sphere
}
Solid {
  translation 0 0.125 1
  children [
    DEF cyl Shape {
      appearance Appearance {
        material Material {
          diffuseColor 0.1803921568627451 0.20392156862745098 0.21176470588235294
        }
      }
      geometry Cylinder {
        height 0.25
        radius 0.1
      }
    }
  ]
  name "solid(26)"
  boundingObject USE cyl
}
